packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -10,6 +10,7 @@ import {
   interpolateString,
   useConfig,
 } from '@openmrs/esm-framework';
+import { displayName } from '@openmrs/esm-utils';
```

Comment:
Now that [O3-3265](https://github.com/openmrs/openmrs-esm-core/pull/1010) exports have been fixed, this import can be simplified.
Import from `@openmrs/esm-framework`